m2cgen/interpreters/interpreter.py

```diff
@@ -5,6 +5,9 @@
 
 class BaseInterpreter:
 
+    with_vectors = False
+    with_linear_algebra = False
```

Comment:
Aren't these 2 attributes interchangeable? I can't find a use case when we may have one without another.

Reply:
Done